### PR TITLE
lisa: Initial Release A14 Official

### PR DESCRIPTION
**Pull request by adrian1234a**

### CODE REVIEW

File: lisa.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "adrian1234a",
```
Comment (gwolf2u) [inline lisa.json:4]: fix indentation

File: lisa.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"device": "Xiaomi 11 Lite 5G NE",
+			"filename": "crDroidAndroid-14.0-20231112-lisa-v10.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/lisa/10.x/crDroidAndroid-14.0-20231112-lisa-v10.0-BETA.zip/download",
+			"timestamp": 1699802338,
+			"md5": "00f4135b98865b197e6404a83a0ce827",
+			"sha256": "429a4118becc47214377f79ab25a99c4ae7d9cf63a897a8d171ae1621796ff40",
+			"size": 1437562307,
+			"version": "10.0",
+			"buildtype": "Beta",
+			"forum": "",
```
Comment (gwolf2u) [inline lisa.json]: missing forum tag

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Comment by adrian1234a** [comment]
now is fine?

**Review by ghost — APPROVED** [review]

**Review by gwolf2u — APPROVED** [review]

**Comment by nondetect** [comment]
Is it possible to install a new build available on Sourceforge via OTA?

**Comment by nondetect** [comment]
> now is fine?

Is it possible to install a new build available on Sourceforge via OTA?

**Comment by nondetect** [comment]
> now is fine?

@adrian1234a  Is it possible to install a new build available on Sourceforge via OTA?

**Pull request merged**